runtime/doc/spell.txt

```diff
@@ -313,10 +313,6 @@ Only the first file is loaded, the one that is first in 'runtimepath'.  If
 this succeeds then additionally files with the name LL.EEE.add.spl are loaded.
 All the ones that are found are used.
 
-If no spell file is found the |SpellFileMissing| autocommand event is
-triggered.  This may trigger the |spellfile.vim| plugin to offer you
-downloading the spell file.
-
```

Comment:
This is already described in the SPELL FILE MISSING section below. Doesn't need to be repeated.